File: templates/matlab/matrix_tensor_monitoring.m
function fig = matrix_tensor_monitoring()
    addpath(fullfile(fileparts(mfilename('fullpath')),'..','..','_utils','matlab'));
    fig = generated_template_figure('band_timeseries', 4401, 'matrix and tensor visualization: monitoring band time series', 'matrix and tensor visualization', 'monitoring band time series');
end

File: templates/python/matrix_tensor_monitoring.py
"""matrix_tensor_monitoring: 矩阵与张量可视化监测带状时序（time-band 模式，合成数据）."""
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '_utils', 'python'))
from generated_patterns import make_template_figure


def make_figure(title='matrix and tensor visualization: monitoring band time series'):
    return make_template_figure('band_timeseries', seed=4401, title=title, domain='matrix and tensor visualization', topic='monitoring band time series')


if __name__ == '__main__':
    fig = make_figure()
    fig.savefig(__file__.replace('.py', '.png'), dpi=150)
